src/main/java/com/rht_labs/Project.java

```diff
@@ -1,5 +1,11 @@
 package com.rht_labs;
```

Comment:
This a general fyi - in java apps - underscores are not used in packages as a convention. This would normally be a package com.rht.labs or probably more normally com.rhc.labs (redhat consulting) main redhat would be com.redhat.xxx

Reply:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html